### PR TITLE
[admin 3/x][fullstack] adds app review page, so we can actually start putting reviews up

### PR DESCRIPTION
**Pull request by saresend**
## What 
* Adds app review page
* Wires up frontend so that reviews actually go through 

## What this means 

Once this ships live and we migrate the production database, we have a *functional* app review dashboard

### CODE REVIEW

File: lib/admin.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { RecyclerViewBackedScrollView } from "react-native";
```
Comment (WilhelmWillie) [inline lib/admin.tsx]: lol wut

Comment (WilhelmWillie) [reply lib/admin.tsx]: lgtm once this is removed

Comment (saresend) [reply lib/admin.tsx]: Oh fuck lmao, vscode must've tried to guess at some imports

Comment (WilhelmWillie) [reply lib/admin.tsx]: bump

Comment (saresend) [reply lib/admin.tsx]: My b, alright we're back time to merge this

**Comment by saresend** [comment]
Alright sorry for going awol, this should be rebased and fixed now, pls take a look and lets get reviews going

**Pull request merged**